### PR TITLE
Updated the start developing guide as suggested in issue #75

### PR DESCRIPTION
**Pull request by adrianmarinwork**
I've updated the content of the developing.mdx file that explains how to start and run the localhost of Infiscal.
The changes are mainly done in the **Set up environment variables** as it's the one that gave me more problems and because the rest it's clear enough already.
I also recommend referring to this file somewhere in the main ReadMe of this repository or as you already have linked, in the [get started contribution guide](https://infisical.com/docs/contributing/overview), because when I started trying to run the localhost I didn't know that this file existed.

If you have any suggestions or changes that you want to make just tell me.

### CODE REVIEW

File: docs/contributing/developing.mdx [hunk 1]
```diff
@@ -16,17 +16,48 @@ cd infisical
...
 ```
+PRIVATE_KEY='oGVv5rThrpZ7WLgQW27chY1cXngr4wLQIZnGfSKgHPk='
+PUBLIC_KEY='ldr6JaC7AY+tun3omGLdE4SWpkJbtVBOI54KfUP53Xc='
+```
+
+2. Mongo variables and site URL
+
+![image](https://user-images.githubusercontent.com/118568289/206792171-3376e3c6-c3ac-4d5d-8776-d78ee089b520.png)
+
+These variables are used to connect the MongoDB and set the URL for the localhost.
```
Comment (mv-turtle) [inline docs/contributing/developing.mdx:41]: Something to also add here is that `MONGO_USERNAME` and `MONGO_PASSWORD` should appear in `MONGO_URL` after `//` in the form of `MONGO_USERNAME:MONGO_PASSWORD`. The recommendation is to just use `root:example` for development purposes though.

@adrianmarinwork do you think this makes sense to be added here?

**Comment by mv-turtle** [comment]
I think the rest looks great! Let me know what you think about my comment, and we can then merge :)

File: docs/contributing/developing.mdx (hunk 1 above)
Comment (adrianmarinwork) [reply docs/contributing/developing.mdx:41]: Yeah, it could be a good thing to pinpoint in the case that the user uses his own username and password.

Comment (adrianmarinwork) [reply docs/contributing/developing.mdx:41]: Done!

Comment (mv-turtle) [reply docs/contributing/developing.mdx:41]: This is perfect!

**Review by mv-turtle — APPROVED** [review]
Thank you so much @adrianmarinwork! This is great!

**Pull request merged**